test/Microsoft.ML.Tests/TextLoaderTests.cs

```diff
@@ -334,17 +600,15 @@ public void CanSuccessfullyTrimSpaces()
         [Fact]
         public void ThrowsExceptionWithPropertyName()
         {
-            Exception ex = Assert.Throws<InvalidOperationException>(() => new Legacy.Data.TextLoader("fakefile.txt").CreateFrom<ModelWithoutColumnAttribute>());
-            Assert.StartsWith($"Field or property String1 is missing {nameof(LoadColumnAttribute)}", ex.Message);
-        }
-
-        [Fact]
-        public void CanSuccessfullyColumnNameProperty()
-        {
-            var loader = new Legacy.Data.TextLoader("fakefile.txt").CreateFrom<ModelWithColumnNameAttribute>();
-            Assert.Equal("Col1", loader.Arguments.Column[0].Name);
-            Assert.Equal("Col2", loader.Arguments.Column[1].Name);
-            Assert.Equal("String_3", loader.Arguments.Column[2].Name);
+            var ml = new MLContext(seed: 1, conc: 1);
+            try
+            {
+                ml.Data.ReadFromTextFile<ModelWithoutColumnAttribute>("fakefile.txt");
+            }
+            catch (Exception ex)
```

Comment:
Can we please use Assert.Throws with the expected exception.

Reply:
Done.